### PR TITLE
RPC Node setup guides for all External Chains

### PR DESCRIPTION
**Pull request by horsovski**
Hey, I wrote the full guides on how to setup RPC nodes for all of the external chains currently supported:

- Avalanche
- Ethereum
- Fantom
- Moonbeam
- Polygon

All of the guides are up to date with the latest Testnet versions and also include links to official documentation. I have also set-up the docsify locally so all of the new changes fit into the existing framework and use the same styling.

Kind Regards,
Rok

chains1
chains2

### CODE REVIEW

File: documentation/docs/validator-zone/external-chains/fantom.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+apt install tmux
+```
+
+Now create a new session called `fantom`:
+
+```bash
+tmux new -s fantom
+```
+
+Once you're inside the newly created tmux session, start the `opera-go`:
```
Comment (aditya-manit) [inline documentation/docs/validator-zone/external-chains/fantom.md:70]: We can use a service  file maybe ? Usually helps with open file limit

File: documentation/docs/validator-zone/external-chains/polygon.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
+# Set up your Polygon Mumbai Testnet node
+-----------
+## Disclaimer
+!> :fire: Please note that the guide below may not always be up to date with the latest version or updates from the project, so make sure you always check the official documentation first and run the latest version of software and required dependencies. If you find a problem, please submit an issue to this repository following the template.
+
+
+## Prerequisites
+- Complete all steps from [Setup with Docker](/setup/setup-with-docker.md) or [Setup with Binaries](/setup/setup-with-binaries.md)
+- Minimum hardware requirements: 4-8+ core CPU , 16-32GB RAM, 100GB+ free storage space.
```
Comment (aditya-manit) [inline documentation/docs/validator-zone/external-chains/polygon.md:9]: Correct me if I am wrong but polygon would take much more than 100GB+ right ?

File: documentation/docs/validator-zone/external-chains/polygon.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+```bash
+eth_rpc_url = <insert Infura or any full node RPC URL to Goerli>
+```
+
+Open the `~/node/bor/start.sh` and add the following flag to start parameters:
+```bash
+--bootnodes "enode://320553cda00dfc003f499a3ce9598029f364fbb3ed1222fdc20a94d97dcc4d8ba0cd0bfa996579dcc6d17a534741fb0a5da303a90579431259150de66b597251@54.147.31.250:30303"
+```
+
+## Start the Polygon services
```
Comment (aditya-manit) [inline documentation/docs/validator-zone/external-chains/polygon.md:134]: Polygon has snapshot for testnets available here: https://snapshots.matic.today/ which can save hours if not days of sync up

File: documentation/docs/validator-zone/external-chains/fantom.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
+# Set up your Fantom Opera node
+-----------
+## Disclaimer
+!> :fire: Please note that the guide below may not always be up to date with the latest version or updates from the project, so make sure you always check the official documentation first and run the latest version of software and required dependencies. If you find a problem, please submit an issue to this repository following the template.
+
+
+## Prerequisites
+- Complete all steps from [Setup with Docker](/setup/setup-with-docker.md) or [Setup with Binaries](/setup/setup-with-binaries.md)
+- Minimum hardware requirements: 4 vCPU+, 100GB+ free storage space.
```
Comment (aditya-manit) [inline documentation/docs/validator-zone/external-chains/fantom.md:9]: Space requirement would be more I believe

File: documentation/docs/validator-zone/external-chains/fantom.md (hunk 1 above)
Comment (horsovski) [reply documentation/docs/validator-zone/external-chains/fantom.md:70]: > We can use a service file maybe ? Usually helps with open file limit

I've used service files on other guides except this one, to diversify a bit but essentially it's down to an user to use his preferred method. I haven't encountered "too many open files" ERR on EVM chains yet but can also add a note.

File: documentation/docs/validator-zone/external-chains/polygon.md (hunk 2 above)
Comment (horsovski) [reply documentation/docs/validator-zone/external-chains/polygon.md:9]: > Correct me if I am wrong but polygon would take much more than 100GB+ right ?

22G     ./.heimdalld/data
54G     ./.bor/data/bor/chaindata

I've used snapshots and I have it in notes as well but not sure why it wasn't included in the merge. Will add the note, thanks.

File: documentation/docs/validator-zone/external-chains/polygon.md (hunk 3 above)
Comment (horsovski) [reply documentation/docs/validator-zone/external-chains/polygon.md:134]: see previous reply.

File: documentation/docs/validator-zone/external-chains/polygon.md (hunk 2 above)
Comment (aditya-manit) [reply documentation/docs/validator-zone/external-chains/polygon.md:9]: Got it - I guess I had downloaded the archive one which was much more than this iirc

Comment (horsovski) [reply documentation/docs/validator-zone/external-chains/polygon.md:9]: Added Polygon fix:
https://github.com/axelarnetwork/axelarate-community/pull/282/commits/1f02df7b593b744b02c044aa17e8c14b888df541

**Review by ggutoski — APPROVED** [review]
This looks great!  Thank you very much!

**Pull request merged**